app/Domain/Integrations/Controllers/IntegrationController.php

```diff
@@ -132,6 +133,8 @@ public function store(StoreIntegrationRequest $request): RedirectResponse
             $this->integrationRepository->save($integration);
         }
 
+        IntegrationCreatedPost::dispatch($integration->id);
```

Comment:
This could also be dispatched inside the repository to have it in a more common/general place?

Reply:
Maybe you could dispatch it inside `EloquentIntegrationRepository::saveTransaction`

Reply:
done